tailwind.config.js

```diff
@@ -10,7 +10,46 @@ module.exports = {
       fontFamily: {
         helvetica: ['Helvetica', 'Arial', 'sans-serif'],
       },
+      screens: {
+        '3xl': '1920px',
+      },
+      colors: {
+        SSW_Red: '#CC4141',
```

Comment:
I think this is how it should look

```suggestion
        ssw: {
            red: "CC4141",
            black: ""
            grey: ""
            }
```

Reply:
done